### PR TITLE
Add wixsite.com and editorx.io

### PR DESCRIPTION
**Pull request by shahata**
* [x] Description of Organization
* [x] Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)

Description of Organization
====

Organization Website: https://www.wix.com

Wix is a website building platform.

Registered users get their own subdomain (e.g. username.wixsite.com or username.editorx.io) on which their sites are hosted.

Reason for PSL Inclusion
====

We'd like to properly isolate each subdomain to prevent global cookie setting on wixsite.com/editorx.io and highlight that each subdomain hosts sites of a different user, unrelated to the others.

DNS Verification via dig
=======

```
dig +short TXT _psl.wixsite.com
"https://github.com/publicsuffix/list/pull/927"
```

```
dig +short TXT _psl.editorx.io
"https://github.com/publicsuffix/list/pull/927"
```

make test
=========

https://gist.github.com/shahata/d1c21c6a7fc30c3f0f2cc9e29fe89add

### CODE REVIEW

**Comment by shahata** [comment]
@dnsguru this is an old PR that I reopened so just pinging it in case it might have been overlooked

**Comment by dnsguru** [comment]
@shahata thx for nudge, we prefer new PR so that the branch is freshest, but that keeps us focused at top of list (I suffer from "scroll-iosis" 😉- the art of focusing at first few lines - to stay efficient w volunteer time).  Back to point... did you recently re-base the pull?  

My bias would be to have that happen, but we're likely safe.  I want to have another reviewer corroborate before I squash/merge.

**Review by dnsguru — APPROVED** [review]
- DNS Validation ok
- Tests ok
- Rationale ok
- _POSSIBLE_ conflict with base - pulling in second reviewer (@sleevi) to ease my concern.

Tentatively **Approved**, pending second opinion from @sleevi

**Comment by shahata** [comment]
@dnsguru thanks for the quick reply! I rebased the PR.

**Comment by dnsguru** [comment]
@sleevi no action needed.  The concern I had was addressed by submitter.

**Review by dnsguru — APPROVED** [review]
Rebase addressed the concern I had.  Fully approved.

**Comment by shahata** [comment]
BTW, what is the way to make sure this gets into Chromium soon? I see @sleevi is the main guy in [there](https://chromium.googlesource.com/chromium/src/+log/master/net/base/registry_controlled_domains/effective_tld_names.dat) 😄

**Comment by sleevi** [comment]
There’s nothing further needed. It should appear in a future release of Chrome in 3-9 months.

**Comment by shahata** [comment]
Yeah what I meant is that I see the list was updated in Chromium few days ago so I hope I didn't miss the train and have to wait additional few months until the next time it is synced.

**Comment by shahata** [comment]
@dnsguru Hi, sorry for nudging, but it's been almost 4 months since this was merged. Any clue when can we expect it to be updated in Chromium?

**Comment by sleevi** [comment]
3-9 months is an accurate reflection. The PSL does not provide any guarantees on if and when browsers will adopt.

**Comment by dnsguru** [comment]
The PSL is essentially a catalog.

Acknowledging that there is no control that this project or the volunteers
has on downstream use of the catalog, I updated the intake and guidance to
parties submitting new additions.

**Pull request merged**
